frontend/src/metabase/setup/components/EmbeddingSetup/steps/ProcessingStep.tsx

```diff
@@ -27,8 +27,9 @@ export const ProcessingStep = () => {
   const setupSettings = useCallback(async () => {
     await updateSettings({
       "embedding-homepage": "visible",
-      "embedding-app-origins-interactive": "*",
       "enable-embedding-interactive": true,
+      "embedding-app-origins-interactive": "localhost:*",
+      "session-cookie-samesite": "none",
```

Comment:
Are we sure we want to use `SameSite=None` as the default? I thought Lax would be a better default option.